raisinbread/RB_files/RB_mri_parameter_form.sql

```diff
@@ -0,0 +1,1574 @@
+ET FOREIGN_KEY_CHECKS=0;
```

Comment:
1. There's a typo
2. Why does this need to disable foreign key checks?

Reply:
1, fix 
2, all the RB sql start with  disable foreign key checks, at the end of this table, it will enable the foreign key again.